### PR TITLE
Adding catalog for Cudo Compute

### PR DESCRIPTION
**Pull request by JungleCatSW**

### CODE REVIEW

File: .github/workflows/update-cudo-catalog.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: "update-cudo-catalog"
```
Comment (Michaelvll) [inline .github/workflows/update-cudo-catalog.yml:1]: How about we separate this auto update action from the catalog? The order of the PR to be merged can be:
1. the `vms.csv`
2. the PR supports Cudo clouds: https://github.com/skypilot-org/skypilot/pull/2975
3. Auto catalog fetching action PR.

File: catalogs/v5/cudo/vms.csv [hunk 2]
```diff
@@ -0,0 +1,81 @@
+InstanceType,AcceleratorName,AcceleratorCount,vCPUs,MemoryGiB,Price,Region,GpuInfo,SpotPrice
+ice-lake-a40_4x1v2gb,A40,1.0,2,4,0.808400,se-stockholm-1,"{'Gpus': [{'Name': 'A40', 'Manufacturer': 'NVIDIA', 'Count': '1.0', 'MemoryInfo': {'SizeInMiB': 49152}}], 'TotalGpuMemoryInMiB': 49152}",0.808400
```
Comment (Michaelvll) [inline catalogs/v5/cudo/vms.csv:2]: Do we have spot instance support on Cudo? If not, it would be better to leave the `SpotPrice` empty.

**Review by Michaelvll — COMMENTED** [review]
Thanks for submitting the catalog for Cudo @CudoVentures! This is very promising. Just left several comments. : )

File: .github/workflows/update-cudo-catalog.yml (hunk 1 above)
Comment (JungleCatSW) [reply .github/workflows/update-cudo-catalog.yml:1]: Okay great Ill sort it.

**Comment by JungleCatSW** [comment]
@Michaelvll I have split this into 2 PRs as recommended

**Review by Michaelvll — APPROVED** [review]
Thanks for the quick update @JungleCatSW! The current catalog looks good to me.
We can leave it to the future, but it seems Cudo also supports CPU instances. Do you think we can support those in the future as well?

**Pull request merged**